get-started-linux.md

```diff
@@ -1,169 +1,195 @@
+The Model Optimizer is a Python\*-based command line tool for importing trained 
+models from popular deep learning frameworks such as Caffe\*, TensorFlow\*, 
+Apache MXNet\*, ONNX\* and Kaldi\*.
 
-You cannot perform inference on your trained model without running the model through the Model Optimizer. When you run a pre-trained model through the Model Optimizer, your output is an Intermediate Representation (IR) of the network. The Intermediate Representation is a pair of files that describe the whole model:
+You cannot perform inference on your trained model without having first run the 
+model through the Model Optimizer. When you run a pre-trained model through the 
+Model Optimizer, it outputs an *Intermediate Representation*, or *(IR)* of 
+the network, a pair of files that describes the whole model:
 
 - `.xml`: Describes the network topology
 - `.bin`: Contains the weights and biases binary data
 
-For more information about the Model Optimizer, refer to the [Model Optimizer Developer Guide](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html). 
+For more information about the Model Optimizer, refer to the 
+[Model Optimizer Developer Guide]. 
 
 ### Model Optimizer Configuration Steps
 
-You can choose to either configure all supported frameworks at once **OR** configure one framework at a time. Choose the option that best suits your needs. If you see error messages, make sure you installed all dependencies.
+You can choose to either configure all supported frameworks at once **OR** 
+configure one framework at a time. Choose the option that best suits your needs. 
+If you see error messages, check for any missing dependencies.
 
-> **NOTE**: Since the TensorFlow framework is not officially supported on CentOS*, the Model Optimizer for TensorFlow can't be configured and ran on those systems.  
+> **NOTE**: The TensorFlow\* framework is not officially supported on CentOS\*, 
+  so the Model Optimizer for TensorFlow cannot be configured on, or run with 
+  CentOS.  
 
-> **IMPORTANT**: The Internet access is required to execute the following steps successfully. If you have access to the Internet through the proxy server only, please make sure that it is configured in your OS environment.
+> **IMPORTANT**: Internet access is required to execute the following steps 
+  successfully. If you access the Internet via proxy server only, please make 
+  sure that it is configured in your OS environment as well.
 
 **Option 1: Configure all supported frameworks at the same time**
 
-1.  Go to the Model Optimizer prerequisites directory:
-```sh
-cd <DLDT_DIR>/model_optimizer/install_prerequisites
-```
-2.  Run the script to configure the Model Optimizer for Caffe,
-    TensorFlow, MXNet, Kaldi\*, and ONNX:
-```sh
-sudo ./install_prerequisites.sh
-```
+1. Go to the Model Optimizer prerequisites directory:
+   `cd <DLDT_DIR>/model_optimizer/install_prerequisites`.
```

Comment:
Why are some code snippets converted to code fragments (from ``` ``` to ` `)?

Reply:
For single-line commands only the single backtick pair is sufficient; for multi-line commands the code fence makes more sense.